_events/2024-03-23-a-sample-event.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: event
+title: "OLS: a training and mentoring program for the open-research-curious"
+description: "Detailed event description"
+date: 2024-03-23
+time: "10:00 AM WAT"
+duration: 2 hours
+location: "Online"
+speakers:
+  - name: "Y.Y"
+  - affiliation: "Open Life Science"
+registration: ""
+type: external
```

Comment:
If external, it should have a link to an external website

Reply:
I thought that might go in the main body of the page. I can also add a new property called `event_link` to the frontmatter.

Reply:
Oh. I think we have different things in mind. For me, `external` was a metadata to say if it links to an external URL and then the `body` is empty.  I feel you think `external` as a metadata to indicate if the event is organized by someone else than OLS.

I think we need to document the possible metadata for an event with a small explanation. That would be easier to discuss. Could you add somewhere? Thanks a lot

Reply:
Found this interesting post about linking to external url: https://thoughtbot.com/blog/external-posts-in-jekyll